src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx

```diff
@@ -1,16 +1,33 @@
 ---
-title: "Login message: Multiple logins found"
+title: "Users with multiple user records"
 tags:
   - Accounts
   - Accounts and billing
   - Account setup
-metaDescription: "When logging into New Relic, you may see a 'multiple logins found' message and be asked to verify your email."
+metaDescription: "Details on features and messages for New Relic users with multiple user records and logins."
 redirects:
 ---
 
 import accountsLoginWithMultipleAccountsFound from 'images/accounts_screenshot-crop_login-with-multiple-accounts-found.png'
 
-If your email is associated with multiple user records, you'll see a `Multiple logins found` message at login, then you'll be asked to verify your email. This might happen if you have access to multiple New Relic organizations, or to multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) in an organization. 
+import accountsUserRecordSwitcher from 'images/accounts_screenshot-crop_user-record-switcher.png'
```

Comment:
@zuluecho9 In this screenshot, it's not clear what menu item leads to the sub menu thing. Maybe update the screenshot to highlight the clicking/mousing path?

Reply:
i will fix that  thanks